Algebra/Discrete_Logarithm.py

```diff
@@ -0,0 +1,22 @@
+# discrete logarithm
+# a^k ≡ b(mod m)a
```

Comment:
Give a brief description about the code/problem

Reply:
Description added.